build.gradle

```diff
@@ -15,7 +15,7 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:7.0.3'
-        classpath 'com.google.gms:google-services:4.3.10'
```

Comment:
The updated broke lint throwing hundreds `Unused resource` errors so I downgraded for now.

Reply:
Let's pull out a separate issue for this.

Reply:
I usually don't file new issues for every dependency that I need to skip during upgrading. Usually if there is something wrong next time it works fine. If during next code freeze the same issue occurs then I will do that.